content/markdown/concepts/emphasis/emphasis.md

```diff
@@ -23,16 +23,16 @@ To bold text, add two asterisks `**` or underscores `__` before and after a word
 **bold text**
 ```
 
-```markdown
-**bold text**
-```
-
 For example:
 
 ```markdown
```

Comment:
Could we change all instances of \```markdown to \```md, per the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#code-blocks)?

Reply:
Totally! Done in [b03c375](https://github.com/Codecademy/docs/pull/656/commits/b03c375e2e651107a2a9fd390c12e4c6a12f7efb)